src/bp/ui-shared/src/Contents/Components/Form/index.tsx

```diff
@@ -320,9 +324,34 @@ const Form: FC<FormProps> = ({
             )}
           </Fragment>
         )
+
+      case 'tag-input':
```

Comment:
can you add a TODO comment as styling mismatches the design specs (which is totally fine at this point)